donate/thunderbird/templates/payment/thank_you.html

```diff
@@ -3,7 +3,7 @@
 {% load i18n util_tags %}
 
 {% block thank_you_description %}
-    <p>{% blocktrans %}Your donation will go right to work protecting the web, keeping it open and available to all.{% endblocktrans %} {% blocktrans with email='give@thunderbird.net' %}We’ve emailed you a donation receipt; if it’s missing, please check your junk/spam folders, then contact us at <a href="mailto:{{ email }}">{{ email }}</a>.{% endblocktrans %}</p>
+    <p>{% comment %}{% blocktrans %}Your donation will go right to work protecting the web, keeping it open and available to all.{% endblocktrans %} {% endcomment %}{% blocktrans with email='thunderbird-donate@mozilla.org' %}We’ve emailed you a donation receipt; if it’s missing, please check your junk/spam folders, then contact us at <a href="mailto:{{ email }}">{{ email }}</a>.{% endblocktrans %}</p>
```

Comment:
As per discussion on Slack with Theo, I think he mentioned for temporary copy change it's better that we use `{% comment %}` instead of removing it completely as it will affect existing localization (those strings are already open for localization)?

/cc @TheoChevalier in case I misunderstood anything

Reply:
From the Slack thread

<img width="524" alt="image" src="https://user-images.githubusercontent.com/2896608/72555457-71bdec00-3851-11ea-8b95-77a0e5f1982f.png">

Reply:
I think some of the localization may get reset in the next deploy from the wagtail l10n project, though I'm not sure I fully understand it; Are these strings indeed already exposed in the Thunderbird pontoon repo?

Reply:
Got it, then yeah, stick with this. Just make the one other string change I suggested and this is good to go.

Reply:
Yup you’re right, Mavis, that’s exactly what I had in mind.
Alan, I guess you’re thinking about the CMS strings? There’s no separate repo for template strings.

The template strings are shared for both Mozilla and Thunderbird and are stored in the files here https://github.com/mozilla/donate-wagtail/tree/master/donate/locale They are indeed fully exposed and 100% complete in most languages. 
Happy to provide further clarifications as needed!